src/templates/static/affiliates/signup.jsx

```diff
@@ -62,13 +62,14 @@ const Signup = () => (
                     items={[
                         { id: 'revenue_share',   text: it.L('Revenue share') },
                         { id: 'turnover',        text: it.L('Turnover') },
+                        { id: 'cpa_eu_only',     text: it.L('CPA (EU only)') },
```

Comment:
why not use our classes to only show this section to EU clients? can you check with Karen?

Reply:
I talked to Karen and she said we can show it to all users, as it says in the title that it's EU only.